backend/src/ticket.ts

```diff
@@ -44,14 +51,14 @@ export function generatePdf(tickets: ITicket[]) {
 
         .font('mp-bold')
         .fill('#000000')
-        .fontSize(21)
-        .text(config.title.toUpperCase(), 60, 30)
+        .fontSize(19)
+        .text(title.toUpperCase(), 60, 30)
 
         .font('mp')
         .text(discount, 60, 30, {align: 'right'})
 
-        .moveUp(0.2)
```

Comment:
feel free to improve - maybe it's just this moveUp that you would like to have back? IMO the upper part of the ticket was getting a bit too crowded, so I took this away, but I trust your visual eye better than mine :p